src/general-components/PaginationFooter/PaginationFooter.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import {PageItem, Pagination} from 'react-bootstrap';
+
+import './paginationFooter.scss'
+
+interface PaginationFooterProps {
+    /**
+     * Anzahl der Seiten, welche zur Auswahl angezeigt werden sollen
+     */
+    pageCount: number
+    /**
+     * Welche Seite gerade ausgewählt ist
+     *
+     * startet bei 1
+     */
+    currentPage: number
+    /**
+     * callback, wenn eine Seite ausgewählt wurde
+     * @param pageNumber nummer der ausgewählten Seite (startet bei 1)
+     */
+    pageChosen: (pageNumber: number) => void
+
+    /**
+     * legt fest ob eingaben möglich sein sollen
+     */
+    disabled?: boolean
+}
+
+/**
+ *
+ */
+class PaginationFooter extends Component<PaginationFooterProps, {}> {
+
+
+    static defaultProps = {
+        currentPage: 1
+    };
+
+    itemClicked(pageNumber: number, e: React.MouseEvent) {
+        e.preventDefault();
+        if (pageNumber !== this.props.currentPage)
+            this.props.pageChosen(pageNumber);
+    }
+
+    nextClicked = (e: React.MouseEvent) => {
+        e.preventDefault();
+        this.props.pageChosen(this.props.currentPage + 1)
+    };
+
+    previousClicked = (e: React.MouseEvent) => {
+        e.preventDefault();
+        this.props.pageChosen(this.props.currentPage - 1)
+    };
+
+    render() {
+                    // clamp active value between 1 and this.props.pageCount
+        let active = this.props.currentPage < 1 ? 1 : (this.props.currentPage > this.props.pageCount ? this.props.pageCount : this.props.currentPage);
+        let items = [];
+                                            // always have at least one page
+        for (let number = 1; number <= (this.props.pageCount < 1 ? 1 : this.props.pageCount); number++) {
+            let isActive = number === active;
```

Comment:
add brackets around boolean expression

Reply:
why?

Reply:
just for optical purpose but it can be removed here, since it is not really that bad